svelte-kit-scss/src/lib/components/IssueSearch/IssueSearch.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, it, expect } from 'vitest';
+import IssueSearchList from './IssueSearchList.svelte';
+import { render, screen } from '@testing-library/svelte';
+import { MOCK_ISSUE_ARRAY } from '$lib/helpers/mocks/issues';
+
+describe('IssueSearch', () => {
+  beforeEach(() => {
+    render(IssueSearchList, {
+      items: MOCK_ISSUE_ARRAY,
+    });
+  });
+
+  describe('List issues', () => {
+    it('should list all issues', () => {
+      const issues = screen.queryAllByTestId('issue-search-list-item');
+      expect(issues.length).toBe(4);
+    });
+  });
+
+  describe('Each issue should contain relevant information', () => {
+    it.each(screen.queryAllByTestId('issue-title'))(
+      'should contain a title that is clickable',
+      (title: HTMLElement) => {
+        expect(title).toHaveProperty('href');
+      }
+    );
+
+    it('should contain an issue number', () => {
+      const issueNumbers = screen
+        .queryAllByTestId('issue-number')
+        .map((number) => number.textContent)
+        .filter(Boolean);
+
+      expect(issueNumbers.length).toBe(4);
```

Comment:
It doesn't really hurt, but you're checking for the length to be four only to follow it up by checking whether the variable equals a 4-length array. You could drop this line and keep the same level of confidence in the code.

Reply:
True

Reply:
In the second case we are testing for the order in which stuff is displayed though @mbicknese

Reply:
Fixed